ci/qa/phplint

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+
+cd $(dirname $0)/../../
+mkdir -p var/qa
+
+# https://github.com/overtrue/phplint
+./vendor/bin/phplint --configuration=ci/qa/phplint.yaml $1
```

Comment:
Out of curiosity, why did you not re-purpose the previous PHP lint? 

Was it hard to get a list of changed files to get this to work?
`sh -c "echo '${changeset.php.spacesep}' | xargs -n 1 -P 4 php -l 1>/dev/null"`

I have little objection to using ta linter tool that can do this work for us. Have you looked at other linting machines that can also lint our other stuff?

Also have a look in the SF console: 

```
 lint
  lint:container                              Ensure that arguments injected into services match type declarations
  lint:twig                                   Lint a template and outputs encountered errors
  lint:xliff                                  Lint an XLIFF file and outputs encountered errors
  lint:yaml                                   Lint a file and outputs encountered errors
```

Reply:
I personally felt that the previous lint was less readable arguments wise and the other projects also use this overtrue/phplint.
If needed  the old one can be implemented again.